usr/share/rear/prep/BACULA/default/400_prep_bacula.sh

```diff
@@ -1,6 +1,19 @@
 ### prepare stuff for BACULA
+if ! test -d "$BACULA_CONF_DIR" ; then
+    test -d "/etc/bacula" && BACULA_CONF_DIR="/etc/bacula"
+    # bacula-enterprise-client uses /opt/bacula/etc
+    test -d "/opt/bacula/etc" && BACULA_CONF_DIR="/opt/bacula/etc"
+fi
+test -d "$BACULA_CONF_DIR" || Error "No BACULA_CONF_DIR"
+if ! test -d "$BACULA_BIN_DIR" ; then
+    BACULA_BIN_DIR="/usr/sbin"
+    # bacula-enterprise-client uses /opt/bacula/bin
+    test -d "/opt/bacula/bin" && BACULA_BIN_DIR="/opt/bacula/bin"
+fi
+export PATH=$PATH:$BACULA_BIN_DIR
 CLONE_GROUPS+=( bacula )
```

Comment:
Only FYI:
With the default `CLONE_ALL_USERS_GROUPS="true"` in default.conf
`CLONE_GROUPS+=( bacula )` is likely no longer needed by default
but for non-default cases it won't hurt to have it still there.